files/bootstrap.sh

```diff
@@ -411,7 +411,7 @@ if [[ "$CONTAINER_RUNTIME" = "containerd" ]]; then
     sudo chown root:root /etc/systemd/system/kubelet.service
     systemctl daemon-reload
     systemctl enable containerd
-    systemctl start containerd
+    systemctl restart containerd
```

Comment:
This is already merged as a part of [this commit](https://github.com/awslabs/amazon-eks-ami/commit/5d109a125ef3876209850d0d4308d1a77ddf601c)